ha/core/controllers/pcs/cluster_controller.py

```diff
@@ -0,0 +1,148 @@
+#!/usr/bin/env python3
+
+# Copyright (c) 2021 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify it under the
+# terms of the GNU Affero General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License along
+# with this program. If not, see <https://www.gnu.org/licenses/>. For any questions
+# about this software or licensing, please email opensource@seagate.com or
+# cortx-questions@seagate.com.
+
+from ha.core.error import HAUnimplemented
+from ha.core.controllers.pcs.pcs_controller import PcsController
+from ha.core.controllers.controller_annotation import controller_error_handler
+
+class PcsClusterController(PcsController):
+    """ Pcs cluster controller to perform pcs cluster level operation. """
+
+    def __init__(self):
+        """
+        Initalize pcs cluster controller
+        """
+        super(PcsClusterController, self).__init__()
+
+    @controller_error_handler
```

Comment:
can we try it to make at class level instead of function?

Reply:
i tried as part of parant class but they are not getting identified in annotation

annotation parse first and then all class and function parse

Reply:
Ok